app/controllers/organisation_relationships_controller.rb

```diff
@@ -18,12 +18,36 @@ def housing_providers
     end
   end
 
+  def add_housing_provider
+    organisations = Organisation.all
+    respond_to do |format|
+      format.html do
+        @organisations = organisations
+        render "organisation_relationships/add_housing_provider", layout: "application"
+      end
+    end
+  end
+
+  def create
+    @resource = OrganisationRelationship.new(child_organisation_id: @organisation.id, parent_organisation_id: related_organisation_params, relationship_type: 0)
+    @resource.save!
+    redirect_to housing_providers_organisation_path
```

Comment:
this should not redirect here unless we use a specific create method like i mentioned here https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/954/files#r1004257319